packages/lab/src/form-field-next/FormFieldControlWrapper.css

```diff
@@ -0,0 +1,7 @@
+.saltFormFieldControlWrapper {
```

Comment:
I'm unsure about whether this component is necessary since all it is doing is giving a default layout when we have multiple inputs in a FormField, which you could also achieve using a horizontal Stack.

How certain are we that this is the only viable layout for these type of controls? If we're pretty sure then I could see this as being a convenient default. But it could be confusing if consumers will occasionally want a different layout.

Reply:
good point, could use Layout components if we aren't too strict about it

Reply:
We talked about this as a solution to a left-aligned left label, where the field with the longest label in a form drives the width of all other field labels. I don't mind how we achieve that though :)

Reply:
this wrapper is actually the container for multi controls

Reply:
Apparently i misunderstood :) There's no requirement for vertical stacking of multiple controls (haven't ever seen anything other than Check/Radio groups... but that would be a single group component, rather than a stack in a form field). Just horizontal grouping.

Reply:
Q for devs is: we either use FlexLayout (as Ben has suggested, until InlineLayout is available which is in backlog) or we provide this Control Wrapper component?